tsl/test/expected/compression_insert-13.out

```diff
@@ -650,6 +650,43 @@ SELECT count(*) FROM trigger_test;
 (1 row)
 
 DROP TRIGGER t3_error ON trigger_test;
+-- test unsupported features are blocked
+-- INSTEAD OF INSERT is only supported for VIEWs
+\set ON_ERROR_STOP 0
+CREATE TRIGGER t4_instead INSTEAD OF INSERT ON trigger_test FOR EACH STATEMENT EXECUTE FUNCTION stmt_trig_info();
+ERROR:  "trigger_test" is a table
+\set ON_ERROR_STOP 1
+-- AFTER INSERT ROW trigger not supported on compressed chunk
+CREATE TRIGGER t4_ar AFTER INSERT ON trigger_test FOR EACH ROW EXECUTE FUNCTION stmt_trig_info();
+\set ON_ERROR_STOP 0
+INSERT INTO trigger_test SELECT '2000-01-01',1,0;
```

Comment:
Does it make sense to add an extra test  here that shows it works as expected if the INSERT does not touch a compressed chunk.? I don't think we have other trigger tests.

Reply:
done